Form-Controls/index.html

```diff
@@ -3,25 +3,65 @@
   <head>
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>My form exercise</title>
-    <meta name="description" content="" />
+    <title>Product Pick</title>
+    <meta name="description" content="A simple form to pick a product (t-shirt)." />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
   </head>
   <body>
     <header>
       <h1>Product Pick</h1>
     </header>
     <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+      <form action="/submit-order" method="post">
+        <div>
```

Comment:
`<div>` is not a semantic tag. Is there a more semantic tag you could use here? (For that matter, what is the purpose of having a container element inside the `<form>` element?